content/sv/docs/faq.md

```diff
@@ -71,7 +71,7 @@ Vi rekommenderar att rapportera sådana sajter till Google Safe Browsing och
 Microsofts SmartScreen-initiativ som har möjlighet att mer effektivt skydda
 användare. Här är Googles rapporteringsadress:
 
-https://safebrowsing.google.com/safebrowsing/report\_badware/
+[https://www.google.com/safebrowsing/report_badware/](https://www.google.com/safebrowsing/report_badware/)
```

Comment:
Was it intentional to revert my URL change I made in pull request #807?

Reply:
Yes, because goldmark parsing was different (compared to blackfriday) I had to update url, I tried to do it in a coherent way between all languages.

Reply:
Aha, you mean the escaping of the underscore? Note that I also changed the URL from www.google.com to safebrowsing.google.com (to save the visitor from one redirect).

Reply:
I've noted to apply that change again in all languages, I'll do it as soon as I find the time, thanks again for that review!